lib/put.js

```diff
@@ -2,12 +2,27 @@ const Node = require('./node')
 
 module.exports = Put
 
-function Put (db, key, value, { batch, del, condition = null, hidden = false, valueBuffer = null }, cb) {
+function putDefaultOptions (opts) {
+  return {
+    condition: null,
+    closest: false,
+    hidden: false,
+    valueBuffer: null,
+    flags: 0,
+    ...opts
```

Comment:
prob breaks our travis for non-spread node support

Reply:
`...` -> `Object.assign`